app/styles/_pipeline.less

```diff
@@ -261,7 +264,7 @@
   .pipeline .pipeline-stage {
     padding-right: (@pipeline-padding * 5.2);
     padding-bottom: (@pipeline-padding + 5);
-    width: 50%;
+    width: calc(100% / 2);
```

Comment:
In this instance, I prefer to just let the Less do the math since the addition of calc doesn't gain us anything, but it's not a big deal either way.